mappings/net/minecraft/world/gen/WorldProvider.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS none/auf net/minecraft/world/gen/WorldProvider
 	FIELD b world Lnone/aiw;
 	FIELD c biomeProvider Lnone/ajn;
+	FIELD d waterEvaporates Z
+	FIELD e noSky Z
```

Comment:
If I was Mojang I might invert the whole thing and use "hasSky" but we can't really do that.